tests/app_test.py

```diff
@@ -98,38 +98,64 @@ def test_connected_endpoint(client, test_user, db_session, auth_header):
     assert res.status_code == 200
 
 
-def test_token_endpoint(client, test_user, db_session):
+def test_token_endpoint_with_default_idp(client, test_user, db_session, auth_header):
```

Comment:
All of the `client.get` requests originally in `test_token_endpoint` seemed to correspond to the same http request on the backend (e.g. when using a debugger, `hasattr(flask.g, "user")` in [`wts/auth.py`](https://github.com/uc-cdis/workspace-token-service/blob/master/wts/auth.py) evaluated to `False` for the first `client.get`, but then evaluated to `True` for each subsequent `client.get`), so in order to isolate `test_token_endpoint_without_auth_header` so it really doesn't have an auth header, I had to put each `client.get` request in its own test.